src/core/ready.js

```diff
@@ -1,6 +1,7 @@
 define( [
 	"../core",
 	"../var/document",
+	"../core/readyException",
```

Comment:
This should be 3.1 only though i.e. minor? Do we have corresponding docs pr/issue?